ReplicaExchange/test/tests.jl

```diff
@@ -0,0 +1,142 @@
+using Test
+
+using Pkg
+Pkg.develop(path = joinpath(pwd(), "..", "..", "OmegaMH"))
+
+using OmegaMH, ReplicaExchange
+
+using Distributions
+using Spec
+import Random
+
+# Gaussian drift proposal
+function propose_and_logratio(rng, state)
+	prop_dist = MvNormal(state, [0.1, 0.1])
+	sample = rand(rng, prop_dist)
+	(sample, 0)
+end
```

Comment:
Omega.jl uses two space indentation - you can use the julia formatter/linter to automate this